Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb

```diff
@@ -1128,8 +1128,8 @@
     "\n",
     "However, it is fun to also show how we can quantify this using the posterior odds (defined in the first section!). For this, we need to calculate the evidence $\\mathcal{Z}(d | M_s)$ and $\\mathcal{Z}(d | M_t)$. They can be estimated by stochastic sampling. However, `bilby_mcmc` is not the right tool (in can in principle, but the alternative is superior for evidence calculation). Instead, we should use the **nested sampling** package `dynesty`\n",
     "\n",
-    "- Rerun your analysis of both models, but use `sampler=\"dynesty\"` in `run_sampler`. You will also need to add configuration arguments: `nlive=500, sampler=\"rwalk\"`)\n",
-    "- Calculate the Bayes factor using `np.exp(result_Mt.log_evidence - result_Mt.log_evidence)`"
+    "- Rerun your analysis of both models, but use `sampler=\"dynesty\"` in `run_sampler`. You will also need to add configuration arguments: `nlive=500, sample=\"rwalk\"`)\n",
```

Comment:
I don't know dynesty enough, but I think the word `sampler` was correct here. They advice to use the `rwalk` sampler, not sample.

Reply:
Yeap, seems you're right. Then it all looks good to me.

Reply:
Thanks for the confirmation.